# :desc: remove transition animations when opening and closing ESC menu.
--- a/panorama/styles/hud_escape_menu.css
+++ b/panorama/styles/hud_escape_menu.css
@@ -6,7 +6,7 @@
 	height: 100%;
 	background-color: #000c;
 	transition-property: opacity;
-	transition-duration: 0.2s;
+	transition-duration: 0s;
 	flow-children: none;
 }
 
@@ -23,7 +23,7 @@
 
 .ShowEscapeMenu #EscapeBackground
 {
-	animation-duration: 0.12s;
+	animation-duration: 0s;
 	animation-timing-function: ease-in-out;
 }
 
@@ -51,7 +51,7 @@
 	opacity: 0.0;
 	transform: translateX( -40px );
 	transition-property: opacity, transform;
-	transition-duration: 0.2s;
+	transition-duration: 0s;
 }
 
 .ShowEscapeMenu #LeftStripe
@@ -131,7 +131,7 @@
 	flow-children: right;
 	overflow: noclip;
 	transition-property: pre-transform-scale2d, background-img-opacity;
-	transition-duration: 0.08s;
+	transition-duration: 0s;
 }
 
 .nav_menu_item:hover
@@ -173,7 +173,7 @@
 	font-family: block;
 	overflow: noclip;
 	transition-property: color, transform;
-	transition-duration: 0.12s;
+	transition-duration: 0s;
 	color: offWhite&aa overflow: noclip;
 	opacity-mask-scale: 160% 160%;
 }
@@ -203,7 +203,7 @@
 	opacity-mask: url("s2r://panorama/images/main_menu/text_speckle_mask_main_psd.vtex") 0.7;
 	opacity-mask-scale: 100% 200%;
 	transition-property: color, background-color, box-shadow, text-shadow, background-img-opacity;
-	transition-duration: 0.1s;
+	transition-duration: 0s;
 	transition-timing-function: ease-in-out;
 }
 
@@ -213,7 +213,7 @@
 	color: offWhite&aa;
 	font-family: block;
 	transition-property: color, background-color, box-shadow, text-shadow;
-	transition-duration: 0.1s;
+	transition-duration: 0s;
 	transition-timing-function: ease-in-out;
 }
 
@@ -267,7 +267,7 @@
 	font-family: block;
 	overflow: noclip;
 	transition-property: color;
-	transition-duration: 0.1s;
+	transition-duration: 0s;
 }
 
 .nav_menu_item.minor .smallIcon
@@ -280,7 +280,7 @@
 	wash-color: offWhite;
 	visibility: collapse;
 	transition-property: opacity, wash-color;
-	transition-duration: 0.3s;
+	transition-duration: 0s;
 }
 
 .nav_menu_item.minor .smallIcon.GearIcon
@@ -319,7 +319,7 @@
 	background-size: contain;
 	opacity: 0;
 	transition-property: transform, opacity;
-	transition-duration: 0.3s;
+	transition-duration: 0s;
 	transition-delay: 0s;
 }
 
@@ -357,7 +357,7 @@
 	vertical-align: middle;
 	transform: translateY( -2px );
 	transition-property: opacity, pre-transform-scale2d, background-image;
-	transition-duration: 0.2s;
+	transition-duration: 0s;
 }
 
 #QuickMute:hover
@@ -388,7 +388,7 @@
 	transform: translateY( -2px );
 	margin-left: 20px;
 	transition-property: opacity, pre-transform-scale2d, background-image;
-	transition-duration: 0.2s;
+	transition-duration: 0s;
 }
 
 .QuickZoo:hover
@@ -475,8 +475,8 @@
 
 .ShowEscapeMenu #changehero
 {
-	animation-name: ChangeHeroIntro;
-	animation-duration: .24s;
+	animation-name: none;
+	animation-duration: 0s;
 	animation-timing-function: ease-in-out;
 }
 
@@ -528,7 +528,7 @@
 	vertical-align: middle;
 	transform: translateY( -2px );
 	transition-property: opacity, pre-transform-scale2d;
-	transition-duration: 0.2s;
+	transition-duration: 0s;
 }
 
 .FeedbackRow #Unstick:hover
@@ -569,7 +569,7 @@
 	opacity: 0.0;
 	transform: translateX( 40px );
 	transition-property: opacity, transform;
-	transition-duration: 0.2s;
+	transition-duration: 0s;
 }
 
 .ShowEscapeMenu #SocialContainer
@@ -671,7 +671,7 @@
 	opacity: 0.0;
 	transform: translateX( 40px );
 	transition-property: opacity, transform;
-	transition-duration: 0.08s;
+	transition-duration: 0s;
 	transition-timing-function: ease-in-out;
 }
 
@@ -687,7 +687,7 @@
 	opacity: 0.0;
 	transform: translateX( 20px );
 	transition-property: opacity, transform;
-	transition-duration: 0.2s;
+	transition-duration: 0s;
 }
 
 #RightSide .FriendsOrPlayersContents TabContents:selected
--- a/panorama/styles/hud_paused.css
+++ b/panorama/styles/hud_paused.css
@@ -72,8 +72,8 @@
 
 #PausedCountdownLabel.ShrinkCountdown
 {
-	animation-name: ShrinkAnimation;
-	animation-duration: 1.0s;
+	animation-name: none;
+	animation-duration: 0s;
 	animation-timing-function: ease-in-out;
 	animation-iteration-count: 1;
 }
@@ -121,7 +121,7 @@
 {
 	text-transform: uppercase;
 	transition-property: transform;
-	transition-duration: 0.5s;
+	transition-duration: 0s;
 	transition-timing-function: ease-in;
 	font-family: block;
 	font-size: 28px;
@@ -146,7 +146,7 @@
 	height: fit-children;
 	opacity: 0;
 	transition-property: opacity;
-	transition-duration: 0.5s;
+	transition-duration: 0s;
 	transition-timing-function: ease-in;
 	background-color: #000000cc;
 	border-radius: 5px;
--- a/panorama/styles/hud.css
+++ b/panorama/styles/hud.css
@@ -7,6 +7,35 @@
 }
 
 .HudCore
+{
+	width: 100%;
+	height: 100%;
+	transition-property: opacity;
+	transition-duration: 0s;
+}
+
+.ShowEscapeMenu .HudCore
+{
+	opacity: 0.0;
+}
+
+.HudTakeoverEnabled .HudCore
+{
+	opacity: 0.0;
+}
+
+.HudHiddenPanel
+{
+	visibility: collapse;
+	opacity: 0.0;
+}
+
+CitadelHud.HideOwnHealth #health_and_abilities_container
+{
+	visibility: collapse;
+}
+
+.RadialsRoot
 {
 	width: 100%;
 	height: 100%;
@@ -14,35 +43,6 @@
 	transition-duration: 0.2s;
 }
 
-.ShowEscapeMenu .HudCore
-{
-	opacity: 0.0;
-}
-
-.HudTakeoverEnabled .HudCore
-{
-	opacity: 0.0;
-}
-
-.HudHiddenPanel
-{
-	visibility: collapse;
-	opacity: 0.0;
-}
-
-CitadelHud.HideOwnHealth #health_and_abilities_container
-{
-	visibility: collapse;
-}
-
-.RadialsRoot
-{
-	width: 100%;
-	height: 100%;
-	transition-property: opacity;
-	transition-duration: 0.2s;
-}
-
 .ShowEscapeMenu .RadialsRoot
 {
 	opacity: 0.0;
@@ -65,12 +65,12 @@
 	height: 100%;
 	opacity: 0.0;
 	transition-property: opacity;
-	transition-duration: 0.0s;
+	transition-duration: 0s;
 }
 
 .HudTakeoverEnabled .HudTakeovers
 {
-	transition-duration: 0.2s;
+	transition-duration: 0s;
 	opacity: 1.0;
 }
 
@@ -110,7 +110,7 @@
 {
 	opacity: 0.0;
 	transition-property: opacity;
-	transition-duration: 0.2s;
+	transition-duration: 0s;
 }
 
 .ShowEscapeMenu #EscapeMenu
